src/pages/community/download.md

```diff
@@ -29,24 +29,25 @@ The downloads for the latest release are distributed via mirror sites.
 Older releases can be downloaded from the Apache archive site.
 All release artifacts should be checked for tampering using GPG or SHA-512.
 
-The latest release of Apache YuniKorn is v0.12.2.
+We publish pre-build docker images for everyone's convenience.
```

Comment:
pre-built?

Reply:
prebuilt as one word according to Cambridge dictionary, will fix on commit.